charts/kibana/templates/kibana-networkpolicy.yaml

```diff
@@ -33,18 +33,22 @@ spec:
           component: ingress-controller
           release: {{ .Release.Name }}
     {{- end }}
-    {{ if .Values.createDefaultIndex }}
+    ports:
+    {{- if .Values.global.authSidecar.enabled  }}
+    - protocol: TCP
+      port: {{ .Values.global.authSidecar.port }}
+    {{- else }}
+    - protocol: TCP
+      port: {{ .Values.ports.http }}
+    {{- end}}
+{{ if .Values.createDefaultIndex }}
+  - from:
```

Comment:
Is this correct? We're adding a completely new `from` block to go along with the old one?

Reply:
The old block was:
```yaml
  ingress:
  - from:
    - podSelector:
        matchLabels:
          tier: nginx
          component: ingress-controller
          release: release-name

    - podSelector:
        matchLabels:
          component: kibana-default-index
          release: release-name
          tier: logging
    ports:
    - protocol: TCP
      port: 5601
```
The new block is
```yaml
  ingress:
  - from:
    - podSelector:
        matchLabels:
          tier: nginx
          component: ingress-controller
          release: release-name
    ports:
    - protocol: TCP
      port: 5601

  - from:
    - podSelector:
        matchLabels:
          component: kibana-default-index
          release: release-name
          tier: logging
    ports:
    - protocol: TCP
      port: 5601
```

Reply:
I wanted to add a new from: when createDefaultIndex is enabled

Reply:
https://github.com/astronomer/astronomer/blob/master/charts/external-es-proxy/templates/external-es-proxy-networkpolicy.yaml

Reply:
with current code without authsidecar 

```
  ingress:
  - from:
    - podSelector:
        matchLabels:
          tier: nginx
          component: ingress-controller
          release: astronomer
    
    - podSelector:
        matchLabels:
          component: kibana-default-index
          release: astronomer
          tier: logging
    ports:
    - protocol: TCP
      port: 5601
```
with current code with  authsidecar  it will fail since the port is using 8084 and requires houston authorization
```
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          network.openshift.io/policy-group: ingress
    
    - podSelector:
        matchLabels:
          component: kibana-default-index
          release: astronomer
          tier: logging
    ports:
    - protocol: TCP
      port: 8084
```

Reply:
our new logic is to segregate the ingress and put a seperate rule to handle that case